### DISCUSSION TITLE
docs: add download/resource page for downloading onyx logos/assets

### DISCUSSION THREAD

**Discussion by larsrickert** [SchwarzIT/onyx#177] [Ideas]
We could create a ressource page inside our VitePress documentation that includes all variants of the onyx logo and icon which can be downloaded.

Benefits:
- Improved branding
- Easy download for onyx consumers / management
- Documentation is the first go-to starting point for all related onyx topics, so we don't need to share the logos via teams etc.

Limitations:
- onyx is licensed under the Apache 2.0 licence so the logo usage is already managed by this licence
- Additionally, we should inform the consumers on the resource page how he is allowed to use the logos

**Comment by patrick246** [comment]
> onyx is licensed under the Apache 2.0 licence so the logo usage is already managed by this licence

You can also release the logos under a different licence, more suitable to assets, as Apache 2.0 mainly focuses on code. That license could also be used for assets like icons in general. A good license could be a creative commons license, e.g. CC BY or CC BY-SA.

**Comment by larsrickert** [comment]
We will talk about this in #699 :)

**Comment by mj-hof** [comment]
Will be added to our resource section in our new showcase
